<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XMLWindow</class>
 <widget class="QWidget" name="XMLWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>824</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 37, 48)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="xmlEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>800</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 63 9pt &quot;Yu Mincho Demibold&quot;;
border-radius: 10px;
color: rgba(162, 198, 208, 200);
margin: 10px;
border: 1px solid  rgba(88, 109, 117, 200);
padding: 15px 10px;
background-color: rgb(36, 46, 57);
selection-background-color: rgb(110, 135, 142);
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="XMLInfoLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 63 8pt &quot;Yu Mincho Demibold&quot;;
font-size: 14px;
color: rgb(110, 135, 142);
background-color: transparent</string>
        </property>
        <property name="text">
         <string>Read Mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="editXMLFileBtn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Mincho Demibold</family>
          <pointsize>-1</pointsize>
          <weight>7</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 63 8pt &quot;Yu Mincho Demibold&quot;;
font-size: 16px;
border: 0.5px solid rgb(70, 70, 70);
background: rgb(110, 135, 142);
border-radius: 10px;
color: rgb(242, 255, 245)</string>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="saveXMLFIleBtn">
        <property name="minimumSize">
         <size>
          <width>125</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Yu Mincho Demibold</family>
          <pointsize>-1</pointsize>
          <weight>7</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 63 8pt &quot;Yu Mincho Demibold&quot;;
font-size: 16px;
border: 0.5px solid rgb(70, 70, 70);
background: rgb(110, 135, 142);
border-radius: 10px;
color: rgb(242, 255, 245)</string>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
